scala/private/rule_impls.bzl

```diff
@@ -590,7 +590,33 @@ def _jar_path_based_on_java_bin(ctx):
     jar_path = java_bin.rpartition("/")[0] + "/jar"
     return jar_path
 
-def _write_executable(ctx, rjars, main_class, jvm_flags, wrapper, use_jacoco):
+def _write_executable(ctx, executable, rjars, main_class, jvm_flags, wrapper, use_jacoco):
+    if (_is_windows(ctx)):
+        return _write_executable_windows(ctx, executable, rjars, main_class, jvm_flags, wrapper, use_jacoco)
+    else:
+        return _write_executable_non_windows(ctx, executable, rjars, main_class, jvm_flags, wrapper, use_jacoco)
+
+def _write_executable_windows(ctx, executable, rjars, main_class, jvm_flags, wrapper, use_jacoco):
```

Comment:
can we comment that use_jacoco is currently ignored and it is a todo to deal with coverage?

Reply:
done